spec/features/users/login_spec.rb

```diff
@@ -62,6 +60,42 @@
       expect(page).to have_content("Welcome Admin, #{@admin.name}!")
     end
 
+    it "cannot login user with incorrect email" do 
+      expect(current_path).to eq("/login")
+
+      fill_in :email, with: "incorrect email"
+      fill_in  :password, with: @user.password
+      click_button "Login"
+
+      expect(current_path).to eq("/login")
+
+      expect(page).to have_content("Incorrect email or password.")
+    end
+
+    it "cannot login user with incorrect password" do 
```

Comment:
Thank you for keeping these tests seperate! I know how easy it is for us to write these all into one test, I like that we are working on making sure every test is only testing one thing (this way we can debug easier and confirm test coverage)

Reply:
Yay, I'm glad you like it this way as well! Agree that it makes debugging much easier.